src/main/java/ru/hogwarts/school/Controller/FacultyController.java

```diff
@@ -22,11 +23,22 @@ public Collection<Faculty> getAll() {
         return facultyService.getAll();
     }
 
-    @GetMapping(params = {"color"})
+    @GetMapping("{id}/students")
+    public ResponseEntity<Collection<Student>> getFacultyStudents(@PathVariable Long id) {
+        Collection<Student> students = facultyService.getFacultyStudents(id);
+        return ResponseEntity.ok(students);
+    }
+
+    @GetMapping ("/{color}")
     public Collection<Faculty> getAllFilteredByColor(@RequestParam String color) {
         return facultyService.getAllByColor(color);
     }
 
+    @GetMapping("/{color}, {name}")
```

Comment:
Так работать не будет :(
Лучше пропиши что-то такое:
```suggestion
    @GetMapping("/filtered")
```